_config/events.yml

```diff
@@ -18,3 +18,32 @@
   location: San Francisco, US
   date: 2016-06-28
   url: https://www.redhat.com/summit/
+
+- eventId: 20161026-eclipseconeurope
+  eventOrganization: EclipseCon Europe
+  location: Ludwigsburg, Germany
+  date: 2016-10-26
+  title: Modeling Avengers - Open Source Technology Mix for Saving the World
+  url: https://www.eclipsecon.org/europe2016/session/modeling-avengers-open-source-technology-mix-saving-world
```

Comment:
It uses some Drools because it uses OptaPlanner, but it might not be even mentioned. It's arguable, but I 'd leave it out (although keeping it in is ok for me too).